python/ray/actor.py

```diff
@@ -720,6 +721,37 @@ def _remote(self, args=None, kwargs=None, **actor_options):
         max_task_retries = actor_options["max_task_retries"]
         max_pending_calls = actor_options["max_pending_calls"]
 
+        if scheduling_strategy is None or not isinstance(
+            scheduling_strategy, PlacementGroupSchedulingStrategy
+        ):
+            if placement_group not in ("default", None):
```

Comment:
None is actually not a default value here, should only check `placement_group != "default"`

Reply:
but if placement_group is set to None, there is no need to warn anything, right?

Reply:
placement_group = None actually has a special meaning: user wants to explicitly disable placement group. They should do it via PlacementGroupSchedulingStrategy as well.